src/main/java/core/creatorclasses/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.creatorclasses;
+
+import core.interfaces.Constants;
+import java.util.Random;
+
+public class ColorSupplier implements Constants {
+    public String getRandomColor() {
+        Random rand = new Random();
+        return COLORS[rand.nextInt(NUMBER)];
```

Comment:
```suggestion
public class ColorSupplier implements Constants {
	private Random rand = new Random();
	
    public String getRandomColor() {
        Random rand = new Random();
        return COLORS[rand.nextInt(NUMBER)];
```
це можна перенести до класу з методу щоб не створювати новий об'єкт при виклику методу